### DISCUSSION TITLE
ask a quest : on version 3.13.3 ,a queue with type quorum can have properties "x-max-priority" ,but why on version 3.13.4 can't do that?

### DISCUSSION THREAD

**Discussion by feiyundu** [rabbitmq/rabbitmq-server#11645] [Questions]
### Is your feature request related to a problem? Please describe.

ask a quest : on version 3.13.3 ,a queue with type quorum can have properties "x-max-priority" ,but why on version 3.13.4 can't do that?

### Describe the solution you'd like

ask a quest : on version 3.13.3 ,a queue with type quorum can have properties "x-max-priority" ,on version 3.13.4  also with this feature.

**Comment by lukebakken** [comment]
Quorum queues do not support priorities. We must have added validation for `x-max-priority` in 3.13.4

@michaelklishin ... anything to add?
